airbyte-integrations/connectors/source-posthog/unit_tests/test_components.py

```diff
@@ -85,42 +86,48 @@ def test_update_cursor(test_name, initial_state, stream_slice, last_record, expe
     [
         ("test_empty_state",
          {},
-         [{'end_time': '2021-01-10T00:00:00.000000+0000',
+         [{'end_time': '2021-01-10T23:59:59.999999+0000',
            'project_id': '2331',
            'start_time': '2021-01-01T00:00:00.000000+0000'},
-          {'end_time': '2021-01-20T00:00:00.000000+0000',
+          {'end_time': '2021-01-20T23:59:59.999999+0000',
            'project_id': '2331',
-           'start_time': '2021-01-10T00:00:00.000000+0000'},
-          {'end_time': '2021-01-30T00:00:00.000000+0000',
+           'start_time': '2021-01-10T23:59:59.999999+0000'},
```

Comment:
posthog has code to align end_time from previous slide to start_time from current slice: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-posthog/source_posthog/components.py#L114